rllib/tests/test_ray_client.py

```diff
@@ -36,45 +37,24 @@ def test_cartpole_lstm(self):
         with ray_start_client_server():
             assert ray.util.client.ray.is_connected()
 
-            config = dict(
```

Comment:
Are these moved because this test's goal is only to test ray_client integration?
Not necessarily convergence?

Reply:
Correct!